dist/demo.html

```diff
@@ -1,10 +1,7 @@
-<meta charset="utf-8">
+<meta charset="utf-8" />
 <title>Cloudinary demo</title>
 <script src="./Cloudinary.umd.js"></script>
 
-<link rel="stylesheet" href="./Cloudinary.css">
-
-
 <script>
-console.log(Cloudinary)
+  console.log(Cloudinary);
```

Comment:
remove console?

Reply:
Please notice that this console.log is in a particular file - an HTML file that can be opened locally, for demoing purposes and so console.log here is for signifying that the `Cloudinary` object is available.